Source/FizzWare.NBuilder/Generators/RandomGenerator.cs

```diff
@@ -72,9 +75,11 @@ public virtual decimal Next(decimal min, decimal max)
 
             int integer = rnd.Next(iMin, iMax);
             int fraction = rnd.Next(0, 4000);
-
+#if DNCORE
+            var separator = CultureInfo.CurrentCulture.NumberFormat.CurrencyDecimalSeparator;

```

Comment:
CultureInfo.CurrentCulture is backwards-compatible, so I don't think we need compiler directives for this.